app/lib/ca/Attributes/Values/TextAttributeValue.php

```diff
@@ -280,11 +280,12 @@ public function htmlFormElement($pa_element_info, $pa_options=null) {
  			$vs_height = trim((isset($pa_options['height']) && $pa_options['height'] > 0) ? $pa_options['height'] : $va_settings['fieldHeight']);
  			$vs_class = trim((isset($pa_options['class']) && $pa_options['class']) ? $pa_options['class'] : '');
 			$vs_element = '';
- 			
- 			if (!preg_match("!^[\d\.]+px$!i", $vs_width)) {
+
+// 			Disabling this as we'll either set the width as px values or use classes for elements
+ 			if (!preg_match("!^[\d\.]+px$!i", $vs_width) && $vs_width) {
```

Comment:
Wait sorry I just re-read it.  It's if it _doesn't_ match the pattern.  Fair enough.